arangod/Utils/CollectionKeysRepository.h

```diff
@@ -108,6 +118,13 @@ class CollectionKeysRepository {
   //////////////////////////////////////////////////////////////////////////////
 
   static size_t const MaxCollectCount;
+
+  ////////////////////////////////////////////////////////////////////////////
+  /// @brief stopped flag, indicating that no more CollectionKeys can be
+  /// stored
+  ////////////////////////////////////////////////////////////////////////////
+
+  bool _stopped;
```

Comment:
this variable is never initialized, so its initial value is UB.
```suggestion
  bool _stopped = false;
```

Reply:
This is an oversight, I will fix immediately!